libparsec/crates/protocol/schema/authenticated_cmds/list_frozen_users.json5

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "major_versions": [
+            4
+        ],
+        "req": {
+            "cmd": "list_frozen_users"
+        },
+        "reps": [
+            {
+                "status": "ok",
+                "fields": [
+                    {
+                        "name": "frozen_users",
+                        "type": "List<UserID>"
+                    }
+                ]
+            },
+            {
+                "status": "author_not_allowed"
+            },
+            {
+                "status": "author_not_found"
```

Comment:
This status is not required: if the author doesn't exist then the authentication will fail and the server will return a HTTP 403